src/res/enkf/enkf_main.py

```diff
@@ -396,26 +373,26 @@ def getEnsembleSize(self) -> int:
 
     def ensembleConfig(self) -> EnsembleConfig:
         """@rtype: EnsembleConfig"""
```

Comment:
Maybe just fix up the rtype docstrings while we are here. They are superseded by type annotations.